graphql-ballerina/errors.bal

```diff
@@ -30,7 +30,7 @@ public type InvalidConfigurationError distinct error;
 public type ServiceHandlingError distinct error;
 
 # Represents an error occurred while a listener operation
-public type ListenerError InvalidConfigurationError|ServiceHandlingError;
+public type ListenerError InvalidConfigurationError|ServiceHandlingError|NotSupportedError;
```

Comment:
We need to make these distinct errors and build hierachy right ?

Something like `type InvalidAccountIdError distinct AccountError;`

https://ballerina.io/learn/by-example/user-defined-error.html

Reply:
Yes, this is tracked by [#990](https://github.com/ballerina-platform/ballerina-standard-library/issues/990)